1001-2000/LIT1936Mashaf.xml

```diff
@@ -1,11 +1,14 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="work" xml:lang="en" xml:id="LIT1936Mashaf">
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Mäṣḥafä ḥǝmamatihu wäzena sǝqlätu wäqǝtlätu läʾǝgziʾǝna</title>
-            <author>Mäbaʾa Ṣǝyon</author>
+            <title xml:lang="gez" xml:id="t1">መጽሐፈ፡ ሕማማቲሁ፡ ወዜና፡ ስቅለቱ፡ ወቅትለቱ፡ ለእግዚእነ፡</title>
+            <title xml:lang="gez" type="normalized" corresp="#t1">Maṣḥafa ḥǝmāmātihu wa-zenā sǝqlatu wa-qǝtlatu la-ʾƎgziʾǝna</title>
+            <title xml:lang="en" corresp="#t1">The Book of the Passion of Our Lord</title>
```

Comment:
A literal translation would be rather "Book of suffering and narration of the crucification and killing of our Lord." @thea-m @eu-genia @DenisNosnitsin1970 - Do you think a more literal translation is more appropriate here. Should Marcin probably encode the more common title "The Book of the Passion of Our Lord" as a parallel title t2 instead?

Reply:
I think it makes sense to have "Book of the passion/suffering and narration of the crucifixion and killing of our Lord" as translation of t1, and "Book of the Passion of Our Lord" as t2. It would be helpful for users to provide a title for Tazkāra madḫǝn as well - it is the usual balance between being as complete as possible and finishing the work in a reasonable amount of time. Since Marcin does this in his free time, it can also be a good solution to provide such additional information ourselves.